packages/lastrev-adapter-contentful/src/contentfulAdapter/contentfulAdapter.ts

```diff
@@ -17,16 +17,19 @@ const Adapter = <T>({
   contentRefTypeText = 'Content reference',
   assetRefTypeText = 'Asset reference',
   contentUrlLookup,
-  skipContentTypes
+  skipContentTypes,
+  maxTraverseDepth = 10
 }: AdapterConfig): Transform => (data) => {
   const parsedEntries: Record<string, ParsedEntry> = {};
 
-  const traverse = (obj: unknown) => {
+  const traverse = (obj: unknown, maxDepth: number = maxTraverseDepth) => {
+    if (maxDepth == 0) return null;
```

Comment:
Is this correct?   If a maxDepth of 0 is passed in, it should return the first level still (i.e. no references pulled in)?

compose.js is mapping each field so a depth of "0" would still return the item, but not seeing a map fn call here.
_Line 67_
```
const out = {
    sys: content.sys,
    fields: {
      ...mapValues(omit(content.fields, rootOmitFields), (field) => traverse(field, include, true))
    }
  };
```

Reply:
@spindle79 - The traverse seen on the `compose.js` file is different from this one and defined on that file with a different logic. 

This traverse depth option is only exposed through the adapter config and not when calling `transform(x)`, so that would be a configuration error. 

That being said I updated the check so it supports `maxTraverseDepth=0` correctly.